tests/system/source/main.d

```diff
@@ -121,8 +124,17 @@ int main (string[] args)
     assert(blocks.length == 1);
     assert(blocks[0] == TestGenesis.GenesisBlock, format!"%s Not using expected TestGenesis.GenesisBlock"(PREFIX));
 
+    size_t empty = 0;
     auto target_height = 42;
-    iota(target_height).each!(h => assertBlockHeightAtleast(h));
+    iota(target_height).each!((ulong h)
+    {
+        if (assertBlockHeightAtleast(h) == 0)
+            empty++;
+        else
+            empty = 0;
+        assert(empty < 3,
```

Comment:
What's the meaning of `3`?

Reply:
Good point, I'll create a constant to describe it.

Reply:
`const LimitOfConsecutiveEmptyBlocks = 3;`